packages/contracts/package.json

```diff
@@ -8,12 +8,13 @@
     "hardhat": "^2.7.1"
   },
   "scripts": {
-    "build:forge": "forge build --root . -c contracts --out forge-artifacts",
+    "build:forge": "forge build --optimize --root . -c contracts --out forge-artifacts",
```

Comment:
have to use these flags because forge doesn't seem to actually recognize the foundry.toml file. I think because it's confused by the dir structure of the repo.

Reply:
seems to work for me without adding the flags (using the latest version of forge):
```bash
/o/o/p/contracts > forge config | rg '(src|out)'
extra_output = []
extra_output_files = []
out = 'forge-artifacts'
    'ds-test/=lib/ds-test/src/',
    'forge-std/=lib/forge-std/src/',
src = 'contracts'
```
Maybe it's an issue with an older version of forge.

Reply:
LOL, turned out I'd installed a very old version of foundry using cargo, which was taking precedence over the foundryup version in my path. We can get rid of all this stuff now.

Reply:
ouch. foundryup should detect that.